.circleci/config.yml

```diff
@@ -605,8 +605,10 @@ jobs:
   publish_npm_package:
     <<: *android_defaults
     steps:
-      - attach_workspace:
-          at: ~/react-native
+      - checkout
```

Comment:
Explicitly checking out here to see if this helps with the "The authenticity of host 'github.com (192.30.253.113)' can't be established." issue.